### DISCUSSION TITLE
Adding priority=true on a next image forces the image to load even if its hidden by css

### DISCUSSION THREAD

**Discussion by duveborg** [vercel/next.js#34248] [Ideas]
### Run `next info` (available from version 12.0.8 and up)

    Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 21.2.0: Sun Nov 28 20:28:41 PST 2021; root:xnu-8019.61.5~1/RELEASE_ARM64_T6000
    Binaries:
      Node: 16.13.1
      npm: 8.1.2
      Yarn: 1.21.1
      pnpm: N/A
    Relevant packages:
      next: 12.0.10
      react: 17.0.2
      react-dom: 17.0.2

### What version of Next.js are you using?

12.0.10

### What version of Node.js are you using?

16

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

AWS

### Describe the Bug

I have a website with one portrait image and one landscape at the top of the page. Only one of them are shown depending on screen size. I want to load these with priority = true but this forces the browser to load both images even though only one is visible

### Expected Behavior

Only the visible image is loaded

### To Reproduce

create two image components with priority=rue and hide one with css, both will be fetched

**Comment by balazsorban44** [comment]
This sounds like a duplicate of this feature request if not word by word, but judging from your intent, you want a solution to solve the art direction problem: https://github.com/vercel/next.js/discussions/26728

A more direct duplicate is https://github.com/vercel/next.js/issues/29612 which has a less active conversation.

Here is our most up-to-date answer that I could find: https://github.com/vercel/next.js/discussions/26728#discussioncomment-949714 Head over to that discussion and give your concern. The more upvote/engagement a feature request gets, the higher the likelihood we are going to look more into it! :pray:
